pkg/reconciler/ingress/config/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
```

Comment:
If not mistaken we only generate a deepcopy func for the Config type under this package (annotation for Store is set to false). Should not we just add an annotation for Config only for example: `// +k8s:deepcopy-gen=true` and omit this file?

Reply:
Yes we omit the other annotation as well at the Store type.

Reply:
Sorry, I didn't understand the advantage other than we can remove `doc.go` but anyway changed it as you suggested.